src/home/TabHome.test.tsx

```diff
@@ -65,7 +65,7 @@ const mockBalances = {
 
 jest.mock('src/statsig', () => ({
   getFeatureGate: jest.fn().mockReturnValue(false),
-  getDynamicConfigParams: jest.fn(() => ({
+  getMultichainFeatures: jest.fn(() => ({
```

Comment:
I have no idea how this can be related to this, but since removing the title - `getMultichainFeatures` started to throw error due to not being mocked 🤷